kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardDefault.xml

```diff
@@ -0,0 +1,404 @@
+		<property name="attributes">
+			<list merge="true">
+				<ref bean="ProcurementCardDefault-creditCardLastFour" />
+				<ref bean="ProcurementCardDefault-cardHolderSystemId"/>
+				<ref bean="ProcurementCardDefault-cardholderUser.principalName"/>
+				<ref bean="ProcurementCardDefault-cardholderUser.name"/>
+				<ref bean="ProcurementCardDefault-organizationCode"/>
+				<ref bean="ProcurementCardDefault-reconcilerGroupId"/>
+				<ref bean="ProcurementCardDefault-reconcilerGroup.id"/>
+				<ref bean="ProcurementCardDefault-reconcilerGroup.name"/>
+				<ref bean="ProcurementCardDefault-cardMonthlyNumber" />
+				<ref bean="ProcurementCardDefault-cardCancelCode"/>
+				<ref bean="ProcurementCardDefault-cardOpenDate"/>
+				<ref bean="ProcurementCardDefault-cardCancelDate"/>
+				<ref bean="ProcurementCardDefault-cardExpireDate"/>
+			</list>
+		</property>
+		<property name="relationships">
+			<list>
+				<dd:relationship objectAttribute="cardholderUser">
+					<dd:primitiveAttribute source="cardHolderSystemId" target="principalId"/>
+					<dd:supportAttribute source="cardholderUser.principalName" target="principalName" identifier="true"/>
+					<dd:supportAttribute source="cardholderUser.name" target="name"/>
+				</dd:relationship>
+				<dd:relationship objectAttribute="reconcilerGroup">
+					<dd:primitiveAttribute source="reconcilerGroup.id" target="id"/>
+					<dd:supportAttribute source="reconcilerGroup.name" target="name" identifier="true"/>
+				</dd:relationship>
+			</list>
+		</property>
+	</bean>
+
+  <bean id="ProcurementCardDefault-creditCardNumber" parent="ProcurementCardDefault-creditCardNumber-parentBean" />
+  <bean id="ProcurementCardDefault-creditCardNumber-parentBean" abstract="true" parent="AttributeDefinition">
+    <property name="name" value="creditCardNumber" />
+    <property name="label" value="Procurement Card Number" />
+    <property name="shortLabel" value="Number" />
+    <property name="maxLength" value="16" />
+    <property name="validationPattern" >
+      <bean parent="AlphaNumericValidationPattern"
+            p:exactLength="16" />
+    </property>
+    <property name="control">
+      <bean parent="TextControlDefinition"
+            p:size="16" />
+    </property>
+    <property name="attributeSecurity">
+      <bean parent="AttributeSecurity">
+        <property name="mask" value="true" />
+        <property name="maskFormatter">
+          <bean parent="MaskFormatterSubString" p:maskCharacter="*" p:maskLength="12" />
+        </property>
+      </bean>
+    </property>
+  </bean>
+
+	<bean id="ProcurementCardDefault-creditCardLastFour" parent="ProcurementCardDefault-creditCardLastFour-parentBean" />
+	<bean id="ProcurementCardDefault-creditCardLastFour-parentBean" abstract="true" parent="AttributeDefinition">
+		<property name="name" value="creditCardLastFour" />
+		<property name="forceUppercase" value="false" />
+		<property name="label" value="Card Number Last Four" />
```

Comment:
KFS 3.0 label for this field is "Procurement Card Number Last Four". You might want to check with Misty on whether this label change is OK.